### PR TITLE
Refactor runner

### PR DESCRIPTION
**Pull request by andreab1997**
We want to massively simplify `runnner.py`:

- In `get_output` we want to avoid the rotations: if needed they should be done afterwards. 
- In `__init__ ` we just need to skip the last part in which `targetpids`, `inputpids`, `targetgrid` and `inputgrid` are collected.
@felixhekhorn @AleCandido

### CODE REVIEW

**Comment by AleCandido** [comment]
> In `get_output` we want to avoid the rotations: if needed they should be done afterwards.

In particular, in `get_output` only the basic three lines of computation and return will survive, keeping it minimal.

In any case, this runner will be completely reworked in the context of #138

**Comment by andreab1997** [comment]
This should be ok now, we just need to fix the tests. Before doing that, can you confirm that this is what we want?

File: src/eko/runner.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-"""This file contains the main application class of eko."""
+"""Main application class of eko."""
```
Comment (AleCandido) [inline src/eko/runner.py:2]: Great, I appreciate :D

**Comment by AleCandido** [comment]
> This should be ok now, we just need to fix the tests. Before doing that, can you confirm that this is what we want? @felixhekhorn @AleCandido

Yep, I confirm :)

**Review by felixhekhorn — APPROVED** [review]

**Review by AleCandido — APPROVED** [review]

**Comment by andreab1997** [comment]
I am merging, thanks for this :)

**Pull request merged**